Problem to compile a deb package of aseba

I'm trying to make a debian sid amd64  package for aseba (after reading compil.linux.md)
Works fine to make the deb package enki and dashel
but I've got a problem with the compilation since the source code of aseba
```
ragnarok@valhalla:~/compilation/aseba/aseba$ debuild -i -us -uc -b
....
[100%] Built target aseba-test-simulator
make[2] : on quitte le répertoire « /home/ragnarok/compilation/aseba/aseba/debian/build »
make[1] : on quitte le répertoire « /home/ragnarok/compilation/aseba/aseba/debian/build »
 fakeroot debian/rules binary
install -d debian/tmp/DEBIAN
install -d debian/tmp/usr/bin
install -d debian/tmp/usr/share/doc/aseba
install -d debian/tmp/usr/share/doc/aseba/example-code
install -d debian/tmp/usr/share/doc/aseba/example-scenarios
cp changelog.md debian/tmp/usr/share/doc/aseba/changelog
cp debian/changelog debian/tmp/usr/share/doc/aseba/changelog.debian
cp debian/copyright debian/tmp/usr/share/doc/aseba
cp targets/playground/examples/*.png debian/tmp/usr/share/doc/aseba/example-scenarios
cp: impossible d'évaluer 'targets/playground/examples/*.png': Aucun fichier ou dossier de ce type
make: *** [debian/rules:27: binary-arch] Error 1
dpkg-buildpackage: erreur: fakeroot debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -rfakeroot -us -uc -ui -i -b failed
```
Effectively, if I look the dir I see it's empty 
```
ragnarok@valhalla:~/compilation/aseba/aseba$ ls -R debian/tmp/
debian/tmp/:
DEBIAN  usr

debian/tmp/DEBIAN:

debian/tmp/usr:
bin  share

debian/tmp/usr/bin:

debian/tmp/usr/share:
doc

debian/tmp/usr/share/doc:
aseba

debian/tmp/usr/share/doc/aseba:
changelog  changelog.debian  copyright  example-code  example-scenarios

debian/tmp/usr/share/doc/aseba/example-code:

debian/tmp/usr/share/doc/aseba/example-scenarios:

```
But, I don't know how to solve this problem, any help is welcome

Thx ;)
(PS sorry for my english... i'm not a pro ;)

--- a/debian/rules
+++ b/debian/rules
@@ -24,11 +24,11 @@ binary-arch:
 	cp changelog.md debian/tmp/usr/share/doc/aseba/changelog
 	cp debian/changelog debian/tmp/usr/share/doc/aseba/changelog.debian
 	cp debian/copyright debian/tmp/usr/share/doc/aseba
-	cp targets/playground/examples/*.png debian/tmp/usr/share/doc/aseba/example-scenarios
-	cp targets/playground/examples/*.playground debian/tmp/usr/share/doc/aseba/example-scenarios
-	cp targets/playground/examples/thymio-default-behaviours.aesl debian/tmp/usr/share/doc/aseba/example-scenarios
-	cp targets/playground/examples/*.aesl debian/tmp/usr/share/doc/aseba/example-code
-	cp targets/challenge/examples/challenge-goto-energy.aesl debian/tmp/usr/share/doc/aseba/example-code
+	cp aseba/targets/playground/examples/*.png debian/tmp/usr/share/doc/aseba/example-scenarios
+	cp aseba/targets/playground/examples/*.playground debian/tmp/usr/share/doc/aseba/example-scenarios
+	cp aseba/targets/playground/examples/thymio-default-behaviours.aesl debian/tmp/usr/share/doc/aseba/example-scenarios
+	cp aseba/targets/playground/examples/*.aesl debian/tmp/usr/share/doc/aseba/example-code
+	cp aseba/targets/challenge/examples/challenge-goto-energy.aesl debian/tmp/usr/share/doc/aseba/example-code
 	strip --strip-unneeded -R .note -R .comment  debian/build/*/*.a
 	strip --strip-unneeded -R .note -R .comment  debian/build/*/*/*.a
 	cd debian/build && $(MAKE) install

